plugins/csv/help.md

```diff
@@ -128,14 +128,14 @@ For example, to extract fields 1, 2, 4, 5, 6 the following fields value can be u
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
 |csv|bytes|None|True|Base64 encoded CSV file|None|ZmllbGQxLCBmaWVsZDIKdmFsdWUxLCB2YWx1ZTIK|
-|fields|string|None|True|Fields to filter|None|f1, f2, f3-f6|
+|fields|string|None|True|Fields to filter|None|f1, f2|
```

Comment:
This change should be reflected in the plugin.spec.yaml file. If you complete it in that file and regenerate it should fill this in as well. Just be careful the regeneration doesn't undo anything you already have.